docs/testplan.md

```diff
@@ -548,6 +548,7 @@ Execute `gravity gc`.
 
   Log forwarder settings
   - [ ] Verify that Creating/Deleting/Editing a log forwarder works.
+  - [ ] Verify that after creating a log forwarder resource, it survive upgrades.
```

Comment:
I don't think this belongs in the UI section to be honest, it's going to be lost here. I would add a separate section for it, or made this test a part of one of the existing upgrade-related scenarios (e.g. one of the things to check after an upgrade).